packages/shared/friends/ensureFriendProfile.ts

```diff
@@ -1,22 +1,9 @@
 import { Avatar } from '@dcl/schemas'
 import { ProfileAsPromise, ProfilesAsPromise } from 'shared/profiles/ProfileAsPromise'
 import { ProfileType } from 'shared/profiles/types'
-import defaultLogger from 'shared/logger'
-import { trackEvent } from 'shared/analytics'
 
-export async function ensureFriendProfile(userId: string): Promise<Avatar | undefined> {
-  try {
-    return ProfileAsPromise(userId)
-  } catch (error) {
-    const message = 'Failed while ensuring friend profile'
-    defaultLogger.error(message, error)
-
-    trackEvent('error', {
-      context: 'kernel#saga',
-      message: message,
-      stack: '' + error
-    })
-  }
+export async function ensureFriendProfile(userId: string): Promise<Avatar> {
```

Comment:
I think that ensureFriendProfile is a better name for ProfileAsPromise if you want to rename it. Otherwise this two functions should be deleted

Reply:
In the spirit of breaking a big PR into smaller pieces, how about we leave this change for a very small refactor PR with only this change in the week?